packages/account-v2/src/containers/POICountrySelector/POICountrySelector.tsx

```diff
@@ -1,17 +1,55 @@
 import React from 'react';
-import { InlineMessage, Text } from '@deriv-com/ui';
+import { Button, Divider, InlineMessage, Text } from '@deriv-com/ui';
 import { CountrySelector } from '../../components/CountrySelector';
 
-export const POICountrySelector = () => (
-    <div className='flex flex-col gap-10'>
-        {/*Do Later: Display message based on the kycAuthStatus API response. */}
-        <Text className='text-xs lg:text-sm' weight='bold'>
-            Your identity verification failed because:
-        </Text>
-        <InlineMessage type='filled' variant='error'>
-            We were unable to verify the identity document with the details provided.
-        </InlineMessage>
-        <Text size='xs'>In which country was your document issued?</Text>
-        <CountrySelector label='Country' name='country' />
-    </div>
-);
+type TPOICountrySelectorProps = {
+    errorStatus?: string;
+    handleNext: () => void;
+    onCountrySelect: (value: string) => void;
+};
+
+export const POICountrySelector = ({ errorStatus, handleNext, onCountrySelect }: TPOICountrySelectorProps) => {
+    const [isValid, setIsValid] = React.useState(false);
+
+    return (
+        <div className='grid h-full'>
+            <section className='flex flex-col gap-16'>
+                {errorStatus && (
+                    <div>
+                        <Text weight='bold' size='sm'>
+                            Your identity verification failed because:
+                        </Text>
```

Comment:
shouldnt we show the reasons of failed idv? Is it covered in a different PR?

Reply:
IDV and Onfido Errors will be added in a separate PR. In this PR, I have added one of the error messages ` We were unable to verify the identity document with the details provided.`